README.md

```diff
@@ -1 +1,106 @@
-# github-actions-deploy-stackstorm
+# Deploy StackStorm
+
+GitHub action to deploy [StackStorm](https://stackstorm.com/) to an AWS VM (EC2).
+
+## Prerequisites
+- An [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) and [Access Keys](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-appendix-sign-up.html)
+- The following secrets should be added to your GitHub actions secrets:
+  - AWS_ACCESS_KEY_ID
+  - AWS_SECRET_ACCESS_KEY
+  - ST2_AUTH_USERNAME
+  - ST2_AUTH_PASSWORD
+
+
+## Example usage
+
+Create `.github/workflow/deploy.yaml` with the following to build on push.
+
+```yaml
+name: Deploy ST2 Single VM with GHA
+
+on:
+  push:
+    branches: [ main ]
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - id: deploy
+      name: Deploy
+      uses: bitovi/github-actions-deploy-stackstorm@main
+      with:
+        aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID}}
+        aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY}}
+        aws_default_region: us-east-1
+        st2_auth_username: ${{ secrets.ST2_AUTH_USERNAME}}
+        st2_auth_password: ${{ secrets.ST2_AUTH_PASSWORD}}
+```
+
```

Comment:
It's worth mentioning which resources (AWS, ST2) this GH Action will rely on and create at a high level if user specifies this example config.

Reply:
good point. added a brief overview